src/components/Modal/Modal.tsx

```diff
@@ -102,7 +102,7 @@ type ModalContentProps = {
    *
    * **Default is `"lg"`**.
    */
-  size?: Extract<Size, 'sm' | 'lg'>;
+  size?: Extract<Size, 'sm' | 'lg'> | 'full';
```

Comment:
I guess were okay with the type here being sort of a hybrid-- some values coming from the `Size` type and the new "full" option being its own thing.

Reply:
Yah, there are a small set of cases where we extend [utility types](https://github.com/chanzuckerberg/edu-design-system/pull/2242/files#diff-766208c965d34c3bf67b6cf8dc749036c694359ca121365e8e7e27c08454383eL13) with component-specific entries. This helps with keeping it clear which values are cross-component, and which are not